suite/tests/api/drdecode_aarch64.c

```diff
@@ -82,6 +82,11 @@ test_mov_instr_addr(void)
     instrlist_encode(GD, ilist, generated_code, true);
     protect_mem(generated_code, gencode_max_size, ALLOW_EXEC | ALLOW_READ);
 
+    // Make sure to flush the cache so v8.0 versions of aarch64 avoid stale icache
```

Comment:
Based on commenting conventions at https://dynamorio.org/page_code_style.html, please use /* */ comments in this C file.